orochi/client.py

```diff
@@ -685,6 +705,12 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            f = open(expanduser("~") + '/.orochi.current_song.log', 'w')
+            f.write(u' '.join((track_name, '|', track_performer)).encode('utf-8').strip())
```

Comment:
Yes, but what if a song or an album contains a | character? Then the format either breaks, or you have to introduce escaping, which introduces complexity (see csv).